user_guide_src/source/libraries/uploaded_files/004.php

```diff
@@ -1,5 +1,5 @@
 <?php
 
 [
-    'avatar' => // UploadedFile instance
-]
+    'avatar' => '...' // UploadedFile instance
+];
```

Comment:
This is explanation of the Uploaded File array.
The last `;` is odd.

And `'...'` is also odd, because it seems like string, but in fact it is an object.

Reply:
I chose to revert those to RST.
They do not seem to be intended to be valid executable PHP code.